tests/tests.js

```diff
@@ -4,42 +4,137 @@ const employee = require('./employee')
+      const formattedprice = pricing.formatPrice(15.1566663)
+      expect(formattedprice).to.equal(15.15)
+    })
+  })
+
+  describe('calculateLTDPrice', () => {
+    it('returns the price of an LTD plan for a single employee without employer contribution', () => {
+      const selectedOptions = {
+        familyMembersToCover: ['ee'],
+        coverageLevel: [{ role: 'ee', coverage: 125000 }],
+      }
+      const price = pricing.calculateLTDPrice(products.longTermDisability, employee, selectedOptions)
+
+      expect(price).to.equal(32.04)
+    })
+
+    it('returns the price for a disability product for an employee without employer contribution', () => {
+      const selectedOptions = {
+        familyMembersToCover: ['ee']
+      }
+      const price = pricing.calculateLTDPrice(products.longTermDisability, employee, selectedOptions)
+
+      expect(price).to.equal(32.04)
+    })
+
+  })
+
+  describe('calculateVolLifePrice', () => {
+    it('returns the toal price for a voluntary life product for a single employee without employer contribution', () => {
+      const selectedOptions = {
+        familyMembersToCover: ['ee'],
+        coverageLevel: [{ role: 'ee', coverage: 125000 }],
+      }
+      const price = pricing.calculateVolLifePrice(products.voluntaryLife, selectedOptions)
+
+      expect(price).to.equal(43.75)
+    })
+    it('returns the price for a voluntary life product for an employee with a spouse without employer contribution', () => {
+      const selectedOptions = {
+        familyMembersToCover: ['ee', 'sp'],
+        coverageLevel: [
+          { role: 'ee', coverage: 200000 },
+          { role: 'sp', coverage: 75000 },
+        ],
+      }
+      const price = pricing.calculateVolLifePrice(products.voluntaryLife, selectedOptions)
+
+      expect(price).to.equal(79)
+    })
   })
 
-  it('returns the price for a voluntary life product for an employee with a spouse', () => {
-    const selectedOptions = {
-      familyMembersToCover: ['ee', 'sp'],
-      coverageLevel: [
-        { role: 'ee', coverage: 200000 },
+  describe('calculateVolLifePricePerRole', () => {
+    it('return the VolLifePrice product for each role of employee', () => {
+      const role = 'ee'
+      const coverageLevel = [{ role: 'ee', coverage: 200000 },]
+
+      const price = pricing.calculateVolLifePricePerRole(role, coverageLevel, products.voluntaryLife.costs)
+
+      expect(price).to.equal(70)
+    })
+    it('return the VolLifePrice product for each role of employee', () => {
+      const role = 'sp'
+      const coverageLevel = [
         { role: 'sp', coverage: 75000 },
-      ],
-    }
-    const price = pricing.calculateProductPrice(products.voluntaryLife, employee, selectedOptions)
+      ]
+
+      const price = pricing.calculateVolLifePricePerRole(role, coverageLevel, products.voluntaryLife.costs)
 
-    expect(price).to.equal(71.09)
+      expect(price).to.equal(9)
+    })
   })
```

Comment:
These tests are effectively testing the same thing. In the first test you've passed in a single role and it worked. In the second test you've passed in a single role and it worked. A better second test would have been to have employee and spouse and to update your `it` sentences accordingly.